src/core/AutomatableModel.cpp

```diff
@@ -51,8 +51,9 @@ AutomatableModel::AutomatableModel(
 	m_range( max - min ),
 	m_centerValue( m_minValue ),
 	m_valueChanged( false ),
-	m_setValueDepth( 0 ),
+	m_oldValue(val),
 	m_hasStrictStepSize( false ),
+	m_nextLink(nullptr),
```

Comment:
```suggestion
	m_nextLink(this),
```
Using `this` to indicate no linked models.